browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2449,8 +2441,18 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			e.stopPropagation();
 		};
 
+		var mouseEnterFunction = function () {
+			builder.map.tooltip.beginShow(div);
+		};
+
+		var mouseLeaveFunction = function () {
+			builder.map.tooltip.beginHide(div);
+		};
+
 		$(controls.button).on('click', clickFunction);
 		$(controls.label).on('click', clickFunction);
+		$(controls.button).on('mouseenter', mouseEnterFunction);
+		$(controls.button).on('mouseleave', mouseLeaveFunction);
```

Comment:
shoultn't we do this on the container - `$(div)` ? so label also will trigger the tooltip?

Reply:
yes, I have tested to div container initially, I was not sure if the mouse will trigger,  I will set back to div again